lib/node.js

```diff
@@ -315,7 +315,7 @@ class NodeClient extends Client {
     if (start == null)
       start = 0;
 
-    assert(typeof start === 'number' || typeof start === 'string');
+    assert(typeof start === 'number' || Buffer.isBuffer(start));
```

Comment:
tl;dr: should we keep the API consistent by accepting the `String` creating the `Buffer` for the users.

https://github.com/handshake-org/hsd/issues/357#issuecomment-583449581

Reply:
These aren't HTTP API endpoints though, they're socket calls. All the others assert Buffers:

https://github.com/handshake-org/hs-client/blob/aa246e2f43c5ab8c3e4cf022c17e17dd4d7aa6e2/lib/node.js#L231-L273